content/en/includes/federation-deprecation-warning-note.md

```diff
@@ -0,0 +1,3 @@
+`Federation V1`, the Kubernetes federation API which reuses the Kubernetes API resources 'as is', is considered alpha and its documentation outdated. 
```

Comment:
~is considered alpha and its documentation outdated.~ never achieved GA status and is no longer under active development.  It is documented here for historical purposes only, and its use is not recommended.

Reply:
okay, will make the change in few min

Reply:
@marun change made, pls review again.